pkg/resources/seccomp-profile/profile.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gadget-seccomp-profile
```

Comment:
Why creating a new namespace? We could reuse the `gadget` namespace.

Reply:
I think it is preferable to use the same `gadget` namespace.

If it makes it easier, instead of a single file `pkg/resources/manifests/deploy.yaml`, we could have:
- `pkg/resources/manifests/namespace.yaml`
- `pkg/resources/manifests/seccomp.yaml`
- `pkg/resources/manifests/rbac.yaml` (sa, role, rolebinding, etc.)
- `pkg/resources/manifests/ds.yaml` (not sure why it was called deploy; there are no deployment, only a daemon set)